src/lsp_data.rs

```diff
@@ -80,6 +87,16 @@ pub mod ls_util {
         }
     }
 
+    pub fn location_from_racer_match(mtch: racer::Match) -> Option<Location> {
+        let source_path = &mtch.filepath;
+
+        if let Some(racer::Coordinate { line, column }) = mtch.coords {
+            Some(location_from_position(source_path, line - 1, column))
+        } else {
+            None
+        }
+    }
+
```

Comment:
Hey, also, someone might want to write a LOLCODE language server in Rust at some point in the future, you never know... 😹😹